deployments/ansible/molecule/custom_vars/windows-verify.yml

```diff
@@ -211,3 +211,105 @@
     - name: Assert fluentd custom_config.conf is used
       assert:
         that: custom_fluentd_config.stat.checksum == td_agent_config.stat.checksum
+
+    - name: Download signalfx-dotnet-tracing-1.0.0-x64.msi
```

Comment:
The files under `molecule` are test files.  This test is just making sure that trying to re-install the same version specified in the test config is a no-op since it should already be installed.